### PR TITLE
integrate bridge apis

### PR DESCRIPTION
**Pull request by rjman-ljm**

### CODE REVIEW

File: packages/app/src/controller/bridge/transaction.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import { Resp } from '@parallel/lib'
+import { Context, Next } from 'koa'
+import { FindOneOptions } from 'typeorm'
+import { BridgeTx } from '../../models/bridge_tx'
+import { BridgeTxStatus } from '../../types/bridge'
+import { BridgeCurrencies } from '../../types/bridge'
+import { BridgeNetworks } from '../../types/bridge'
```
Comment (BruceChar) [inline packages/app/src/controller/bridge/transaction.ts:7]: merge in one import

File: packages/app/src/controller/bridge/transaction.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            amount: item.amount,
+            currency: BridgeCurrencies[tokenId],
+        }
+    })
+    ctx.body = Resp.Ok({
+        totalCount,
+        pendingBridgeTxList,
+    })
+
+    return next
```
Comment (BruceChar) [inline packages/app/src/controller/bridge/transaction.ts:43]: next is a callback,  return next()

File: packages/app/src/controller/bridge/transaction.ts (hunk 1 above)
Comment (rjman-ljm) [reply packages/app/src/controller/bridge/transaction.ts:7]: sure

File: packages/app/src/routers/bridge/pending-tx.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import Router from 'koa-router'
+import { getPendingTxListBySs58Address } from '../../controller/bridge/transaction'
+
+const R = new Router()
+
+/**
+ * @api {get} /bridge/pending-transaction/:ss58-address getPendingTxListBySs58Address
```
Comment (BruceChar) [inline packages/app/src/routers/bridge/pending-tx.ts:7]: Add a version path to keep router consitence, e.g. /bridge/v1
And it's better to use plural form for RESTFUL resource，e.g. pending-transactions

File: packages/app/src/controller/bridge/transaction.ts [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+export async function getPendingTxStatus(ctx: Context, next: Next) {
+    ctx.body = Resp.Ok({ status: 'Alive' })
+    return next
```
Comment (BruceChar) [inline packages/app/src/controller/bridge/transaction.ts:48]: next()

File: packages/app/src/routers/bridge/pending-tx.ts (hunk 3 above)
Comment (rjman-ljm) [reply packages/app/src/routers/bridge/pending-tx.ts:7]: 🚀

**Review by BruceChar — APPROVED** [review]

**Pull request merged**